lib/form1010_ezr/service.rb

```diff
@@ -28,6 +28,32 @@ def initialize(user)
       @user = user
     end
 
+    def self.veteran_initials(parsed_form)
```

Comment:
I had to make this method a class-level method as well because `log_submission_failure_to_sentry` calls it.